bigtable/tests/system.py

```diff
@@ -102,6 +105,18 @@ def setUpModule():
 
         EXISTING_INSTANCES[:] = instances
 
+        # Clean up undeleted instances from prior systest run if any
+        def age_in_hours(micros):
+            return (datetime.datetime.utcnow().replace(tzinfo=UTC) - (
+                _datetime_from_microseconds(micros))).total_seconds() // 3600
+        INSTANCE_TO_CLEAN = [
+            instance for instance in EXISTING_INSTANCES if (
+                LABEL_KEY in instance.labels.keys() and
+                age_in_hours(int(instance.labels[LABEL_KEY]))) >= 2]
+        if INSTANCE_TO_CLEAN:
+            for instance in INSTANCE_TO_CLEAN:
+                instance.delete()
+
```

Comment:
I like adding the labels, but would prefer *not* cleaning instances during the system test run:  we lose the ability to figure out why the leak happened.